lib/postgres/postgres-pr/connection.rb

```diff
@@ -60,7 +60,7 @@ def initialize(database, user, password=nil, uri = nil)
     uri ||= DEFAULT_URI
 
     @transaction_status = nil
-    @params = {}
+    @params = { 'username' => user, 'database' => database }
```

Comment:
These params are used to keep track of information that could otherwise be lost between switching modules and datastore options. I use the database param to show the user which DB their session is currently connected to in the interactive prompt, and it is the only DB we can dump when using the schemadump module with a session. Username is being used in the second PR for the hashdump module. Let me know if different implementation would be expected instead 👍